src/openassetio-python/package/openassetio/pluginSystem/PythonPluginSystem.py

```diff
@@ -215,3 +256,12 @@ def __load(self, path):
         module.plugin.__file__ = path
 
         self.register(module.plugin, path)
+
+    def __log_exc(self, ex):
+        """
+        Logs information about the supplied exception into the logger
+        with appropriate formatting.
+        """
+        for frame in traceback.format_tb(ex.__traceback__):
+            for line in frame.splitlines():
```

Comment:
[fix] I don't think we should prefix each line of the stack trace. 

The example that springs to mind (from my experience) is centralised logging, e.g. the [ELK stack](https://www.elastic.co/what-is/elk-stack) where individual log entries are database records in their own right.  It can be a pain when you filter for a specific multi-line message if the lines aren't grouped under the same record.

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/95d33bb5a32498dabc7446caab65d02e370fb880)?

Reply:
(An elk)